superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -744,11 +744,11 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
           engine_information,
         },
       });
-    }
```

Comment:
is this PR changing indentation/block scope?

Reply:
yea i moved it inside of the if so i have context for `engine`